src/main/java/tracko/logic/commands/order/AddOrderCommand.java

```diff
@@ -35,7 +35,8 @@ public class AddOrderCommand extends MultiLevelCommand {
 
     public static final String MESSAGE_USAGE_2 = "To add an item and its quantity to the order being created: \n"
         + CliSyntax.PREFIX_ITEM + "ITEM_NAME "
-        + CliSyntax.PREFIX_QUANTITY + "ITEM_QUANTITY \n"
+        + CliSyntax.PREFIX_QUANTITY + "ITEM_QUANTITY "
+        + CliSyntax.PREFIX_DESCRIPTION + "ITEM_DESCRIPTION \n"
```

Comment:
I was wondering whether the orders require the item description because the order tabs in the UI might get too long if the item has a long description. These are just my thoughts though! Let me know what you think.

Reply:
Ah, you have a good point! I've made the edit!